packages/experimental/Tabs/src/useTabsItem.ts

```diff
@@ -53,7 +59,7 @@ export const useTabsItem = (props: TabsItemProps): TabsItemInfo => {
     if (itemKey == info.selectedKey) {
       info.updateSelectedTabsItemRef && componentRef && info.updateSelectedTabsItemRef(componentRef);
     }
-  }, []);
+  }, [componentRef, info, itemKey]);
```

Comment:
post commit review: Based on the comment I'm pretty sure the dependency array is meant to be empty, since the intent is to run it once post-render.